internal/jobcontainers/jobcontainer.go

```diff
@@ -126,18 +140,39 @@ func Create(ctx context.Context, id string, s *specs.Spec) (_ cow.Container, _ *
 		}
 	}()
 
-	sandboxPath := fmt.Sprintf(sandboxMountFormat, id)
-	if err := mountLayers(ctx, id, s, sandboxPath); err != nil {
-		return nil, nil, errors.Wrap(err, "failed to mount container layers")
-	}
-	container.sandboxMount = sandboxPath
+	// Check if we support file binding once to avoid needing to stat for the dll on
+	// every container creation.
+	checkBindSupportOnce.Do(func() {
+		bindDLL := `C:\windows\system32\bindfltapi.dll`
+		if _, err := os.Stat(bindDLL); err == nil {
+			fileBindingSupport = true
+		}
+	})
 
-	layers := layers.NewImageLayers(nil, "", s.Windows.LayerFolders, sandboxPath, false)
-	r.SetLayers(layers)
+	// If bind mount support is available on the host, there's a lot of new functionality we
+	// can make use of that improves the UX for volume mounts and where the containers rootfs
+	// shows up on the host. You can do per silo bindings so every container can have a static
+	// path for its rootfs location and avoid having to make liberal use of the
+	// CONTAINER_SANDBOX_MOUNT_POINT environment variable.
+	if fileBindingSupport {
```

Comment:
Is there a comment somewhere that explains specifically what are the differences in configuration based on if bindlft is/isn't supported? I would like to see that somewhere if it doesn't exist already.

Reply:
Not not a comprehensive one at least, they're split around near all the fileBindSupport checks like this. I can add!

Reply:
Added a list of changes having bindflt brings. Not sure it's in a great spot, let me know what you think

Reply:
The jobcontainers package could do with a package doc that's pretty in depth.. (or even a small readme probably like we have for /internal/guest)